igvm/hypervisor.py

```diff
@@ -1149,9 +1149,12 @@ def log_migration(self, vm: VM, operator: str) -> None:
         :param operator: plus for migration to HV, minus for migration from HV
         """
 
-        cpu_usage_vm = self.estimate_vm_cpu_usage(vm)
-        timestamp = int(time())
-        log_entry = '{} {}{}'.format(timestamp, operator, round(cpu_usage_vm))
+        cpu_usage_vm = round(self.estimate_vm_cpu_usage(vm))
+        if cpu_usage_vm == 0:
+            return
+
+        timestamp = int(time() * 1000)
```

Comment:
Why not use the opportunity and have a format both human and machine readible, like `datetime.now().isoformat()` ?